packages/server/src/arpa_reporter/lib/audit-report.js

```diff
@@ -125,22 +132,29 @@ async function getProjectSummaryGroupedByProjectRow(data) {
             `${reportingPeriodEndDate} Total Expenditures for Awards Greater or Equal to $50k`,
             `${reportingPeriodEndDate} Total Aggregate Obligations`,
             `${reportingPeriodEndDate} Total Obligations for Awards Greater or Equal to $50k`,
-        ].forEach((columnName) => { row[columnName] = 0; });
+        ].forEach((columnName) => { projectSummaryV2Row[columnName] = 0; });
     });
 
-    row['Capital Expenditure Amount'] = 0;
+    projectSummaryV2Row['Capital Expenditure Amount'] = 0;
 
     // set values in each column
     records.forEach(async (r) => {
+        // for project summaries v2 report
         const reportingPeriodEndDate = reportingPeriods.filter((reportingPeriod) => r.upload.reporting_period_id === reportingPeriod.id)[0].end_date;
-        row[`${reportingPeriodEndDate} Total Aggregate Expenditures`] += (r.content.Total_Expenditures__c || 0);
-        row[`${reportingPeriodEndDate} Total Aggregate Obligations`] += (r.content.Total_Obligations__c || 0);
-        row[`${reportingPeriodEndDate} Total Obligations for Awards Greater or Equal to $50k`] += (r.content.Award_Amount__c || 0);
-        row[`${reportingPeriodEndDate} Total Expenditures for Awards Greater or Equal to $50k`] += (r.content.Expenditure_Amount__c || 0);
-        row['Capital Expenditure Amount'] += (r.content.Total_Cost_Capital_Expenditure__c || 0);
+        projectSummaryV2Row[`${reportingPeriodEndDate} Total Aggregate Expenditures`] += (r.content.Total_Expenditures__c || 0);
+        projectSummaryV2Row[`${reportingPeriodEndDate} Total Aggregate Obligations`] += (r.content.Total_Obligations__c || 0);
+        projectSummaryV2Row[`${reportingPeriodEndDate} Total Obligations for Awards Greater or Equal to $50k`] += (r.content.Award_Amount__c || 0);
+        projectSummaryV2Row[`${reportingPeriodEndDate} Total Expenditures for Awards Greater or Equal to $50k`] += (r.content.Expenditure_Amount__c || 0);
+        projectSummaryV2Row['Capital Expenditure Amount'] += (r.content.Total_Cost_Capital_Expenditure__c || 0);
+
+        // for kpi report
+        const currentPeriodExpenditure = r.content.Current_Period_Expenditures__c || 0;
+        kpiRow['Number of Subawards'] += (r.type === 'awards50k');
```

Comment:
I think you might need to tweak this slightly:

```suggestion
        kpiRow['Number of Subawards'] += (r.type === 'awards50k') ? 1 : 0;
```

Reply:
Initially I had that but thought it was redundant since `r.type === 'awards50k'` should result in 1 or 0 for true/false. For example:
```
var hello = 0;
hello += ('1' === '1'); // hello = 1
```

Reply:
Oh wow, I actually didn't know javascript would coerce true and false to 1 and 0, if you add it like this!

In that case you're right.  This works as is.  I slightly prefer something more explicit, but I don't want to block the change!